hw/ip/padctrl/fpv/padctrl_fpv.core

```diff
@@ -11,7 +11,6 @@ filesets:
       # note: this is an example config which may differ
       # from a particular top-level config
       - lowrisc:ip:padctrl
-      - lowrisc:prim:prim_pkg
     files:
       - tb/padctrl_fpv.sv
```

Comment:
This file is still using the `prim_pkg` package, so you either need to remove the package there as well, or keep the dependency.

Reply:
Thank you for the explanation Philipp. In FPV your "dynamic prim dependency" for fusesoc does not have any issues with the current FPV implementation. I just want to give this fancy improvement a try :P. 
I will add back this prim_pkg for the consistence of the rest of the FPV core files.

Reply:
Yes this is correct, @cindy. The `prim_pad_wrapper` does not expose an `Impl` parameter anymore now. That wrapper is autogenerated and the `Impl` parameter is set internally. So you can safely delete that line there.